test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -138,42 +138,32 @@ Describe "Format-Table DRT Unit Tests" -Tags "CI" {
 
 		It "Format-Table with No Objects for End-To-End should work"{
 				$p = @{}
-				$result = $p|Format-Table -Property "foo","bar"|Out-String
+				$result = $p | Format-Table -Property "foo","bar" | Out-String
 				$result | Should BeNullOrEmpty
 		}
 
 		It "Format-Table with Null Objects for End-To-End should work"{
 				$p = $null
-				$result = $p|Format-Table -Property "foo","bar"|Out-String
+				$result = $p | Format-Table -Property "foo","bar" | Out-String
 				$result | Should BeNullOrEmpty
 		}
 
-		#pending on issue#900
-		It "Format-Table with single line string for End-To-End should work" -pending{
+		It "Format-Table with single line string for End-To-End should work" {
 				$p = "single line string"
-				$result = $p|Format-Table -Property "foo","bar"|Out-String
-				$result | Should BeNullOrEmpty
+				$result = $p | Format-Table -Property "foo","bar" -Force | Out-String
+				$result.Replace(" ","").Replace([Environment]::NewLine,"") | Should BeExactly "foobar------"
 		}
 
-		#pending on issue#900
-		It "Format-Table with multiple line string for End-To-End should work" -pending{
+		It "Format-Table with multiple line string for End-To-End should work" {
 				$p = "Line1\nLine2"
```

Comment:
shouldn't this be 
```
$p = "Line1`nLine2"
```
This looks like it was migrated incorrectly from c#

Reply:
Yes, the escaped newline is wrong, but I'm actually not sure what this test is supposed to be validating.  The multiline doesn't have an effect on format-table.  Is that the test?

Reply:
I believe that is the test, but I'm having trouble reconciling the title of the test with the code. If we want to check multiple line strings in format-table, I see 2 tests which are interesting:
```
[pscustomobject]@{ foo = "line1`nline2"; bar = 2 }| format-table -prop foo,bar -wrap | out-string -str | where-object {$_}
foo   bar
---   ---
l1      1
l2
```
and
```
[pscustomobject]@{ foo = "line1`nline2"; bar = 2 }| format-table -prop foo,bar | out-string -str | where-object {$_}
foo   bar
---   ---
l1...   1
```
but I didn't go back and look at all the tests in this file to see if those cases are covered.
certainly it's not a blocking issue, and the test has some utility now as is